Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -154,14 +154,17 @@ function InvokeGetRestMethod
     $request = $sonarQubeHostUrl + $query;
     
     if ($useAuth)
-    {
-      
+    {      
        $authHeader = CreateBasicAuthHeaderFromEndpoint
-       $allheaders = @{Authorization = $authHeader}   
-    }  
 
-    $response = Invoke-RestMethod $request -Method Get -Headers $allheaders
+       if (![String]::IsNullOrWhiteSpace($authHeader))
```

Comment:
This is the actual fix - do not add an empty Auth header as it will cause the PowerShell system function InvokeRestMethod to throw a Null Ref exception (still trying to raise a bug on them for this)